libs/ui/elements/src/lib/pagination/pagination.component.html

```diff
@@ -0,0 +1,36 @@
+<div style="width: 100%; height: 4rem; display: flex; justify-content: center">
+  <div
+    class="capitalize text-white flex justify-center items-center p-3 rounded-lg mr-3 cursor-pointer"
+    style="background-color: #8cc630"
+    translate
+  >
+    datahub.search.nextPage
+  </div>
+  <div style="display: flex; align-items: center" translate>
+    <span translate>datahub.search.page</span>
+    <span class="border border-gray-500 rounded p-4">{{ currentPage }}</span>
+    <span class="m-3" translate> datahub.search.pageOn </span> {{ nPages }}
+  </div>
+
+  <mat-icon>
+    <button
+      (click)="previousPage()"
+      id="navigate_previous"
```

Comment:
Don't use id for the tests, at least a class.
But you should use css selector on button and take the correct one in the list.